js/components/image.js

```
@@ -291,14 +290,16 @@
 
       this.validateValue();
 
-      getPicture.then(function onSelectedPicture(img) {
-        const imgBase64Url = 'data:image/jpeg;base64,' + img;
+      getPicture.then(function onSelectedPicture(imgBase64Url) {
+        imgBase64Url = (imgBase64Url.indexOf('base64') > -1)
+          ? imgBase64Url
+          : 'data:image/jpeg;base64,' + imgBase64Url;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)